### PR TITLE
New system find-db for 4.3 (gfx900/56, gfx906/60, gfx908/120, gfx1030/36)

### PR DESCRIPTION
**Pull request by cderb**
This is #973 ported for `develop`.

----
find db generated using:
http://compute-artifactory.amd.com/artifactory/rocm-osdb-deb/compute-rocm-rel-4.3-16/
MIOpen branch release/rocm-rel-4.3-staging

900_56 906_60 908_120 1030_36

### CODE REVIEW

**Comment by atamazov** [comment]
@JehandadKhan Is it the same as #973 (same files, same contents of files), but for merging to `develop`? I plan to port #973 to `develop` after merging it, so it would be handy. Otherwise I'll need to create a new branch on `develop`, cherry-pick #973 there and then create a separate PR anyway.

**Comment by JehandadKhan** [comment]
@atamazov Its the same file contents. We can use this PR to update develop.

**Comment by atamazov** [comment]
On hold until #973 is merged.

**Review by atamazov — APPROVED** [review]

**Pull request merged**
